.gitignore

```diff
@@ -1,5 +1,8 @@
 .idea/
 vendor/
 build/
-docker-compose.yml
+.env
```

Comment:
.env heb je verwijderd toch?

Reply:
Ehh ja. Dat was toch conform de afspraken?

Reply:
Maar dan hoeft ie niet in de .gitignore?

Reply:
Oooooohhh... Je hebt em wel staan lokaal, maar je wilt em niet in je repo 😅 Dat is ook het idee van .gitignore 🤦🏻‍♂️ 

Ik ga nog even wat koffie zetten denk ik 😆 ☕

Reply:
:laughing: